api/app/Http/Controllers/OrderController.php

```diff
@@ -22,10 +24,21 @@ public function index()
      */
     public function store(StoreOrderRequest $request)
     {
+
+        $userId = Auth::id();
+
         $validatedData = $request->validated();
+        $category = Category::firstOrCreate(
+            ['label' => $validatedData['category_label']],
+        );
+        $validatedData['category_id'] = $category->id;
 
+        $validatedData['user_id'] = $userId;
```

Comment:
No need to create new variable if used once
```suggestion

        $validatedData = $request->validated();
        $category = Category::firstOrCreate(
            ['label' => $validatedData['category_label']],
        );
        $validatedData['category_id'] = $category->id;

        $validatedData['user_id'] = Auth::id();
```